pkg/lib/config/config.go

```diff
@@ -273,6 +280,23 @@ func (c *AppConfig) validateCustomAttribute(ctx *validation.Context) {
 	}
 }
 
+func (c *AppConfig) validateNFTCollections(ctx *validation.Context) {
+	if c.NFT == nil || c.NFT.Collections == nil {
+		return
+	}
+
+	if len(c.NFT.Collections) > 0 {
+		for i, collection := range c.NFT.Collections {
+			_, err := web3.ParseContractID(collection)
+
+			if err != nil {
+				ctx.Child("nft", "collections", strconv.Itoa(i)).EmitError("invalid contract ID", nil)
```

Comment:
You can introduce a new format to do this automatically. See pkg/lib/config/http.go:8 and pkg/util/validation/formats.go for an example.